CollAction/Views/Home/FAQ.cshtml

```diff
@@ -7,9 +7,8 @@
 }
 
 @section styles {
-    <link rel="stylesheet" href="~/css/faq.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
```

Comment:
Why still using the whole fontawesome stylesheet here instead of wrapping the icons in a React component like the rest?

Reply:
That’s ok